orc8r/lib/go/protos/marshaler_test.go

```diff
@@ -155,9 +156,17 @@ func TestMconfigMarshal(t *testing.T) {
 	assert.NotNil(t, pdAny)
 	pdJson, err := protos.MarshalMconfigToString(pdAny)
 	assert.NoError(t, err)
+	pipelinedExpectedVal = strings.ReplaceAll(pipelinedExpectedVal, " ", "")
```

Comment:
Minor comment: It would be better readable if the string normalization would be extracted into a `stripWhitespace` function